app/routes/news.mdx

```diff
@@ -22,6 +22,34 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2023
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'October 27, 2023'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>New Swift-BAT/GUANO and IceCube Notices Available via GCN Kafka</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        The Swift-BAT/GUANO, IceCube, and GCN teams are pleased to announce the availability of new GCN notice types via the new GCN in JSON format.  JSON format notices can be streamed via [Kafka](/quickstart). [Self-service email subscription](/user/email) will be available soon.  These new notice types are *not* available via GCN Classic.
+
+        [**Swift-BAT/GUANO:**](/missions/swift#json-serialized-gcn-notice-types-in-gcn-kafka) gcn.notices.swift.bat.guano
+
+        - *Summary:* The GUANO system is a family of gamma-ray transient search pipelines that supplement the onboard triggering capabilities of Swift/BAT, significantly enhancing the sensitivity and sky area covered by the instrument.
```

Comment:
As worded, it sounds like GUANO runs onboard, which it does not.

Reply:
@Tohuvavohu

Reply:
How about: "GUANO is a ground-based re-analysis of data from Swift BAT, significantly enhancing the sensitivity and sky area covered as compared to Swift's onboard triggering."

Reply:
remove 're'

Reply:
```suggestion
        - *Summary:* GUANO is a ground-based analysis of data from Swift BAT, significantly enhancing the sensitivity and sky area covered as compared to Swift's onboard triggering.
```